Sources/MarathonCore/ScriptManager.swift

```diff
@@ -242,6 +254,32 @@ internal final class ScriptManager {
 
         return scriptFolder
     }
+    
+    @discardableResult private func createTestsFolderIfNeeded(for script: Script, file: File) throws -> Folder {
+        if (try? script.folder.file(named: "OriginalTestsFile")) == nil {
```

Comment:
Awesome—should I update the syntax in `createFolderIfNeededForScript(...)` as well? Or leave that for another PR?